recipes-kernel/linux/linux-toradex_5.4-2.3.x.bb

```diff
@@ -0,0 +1,32 @@
+require recipes-kernel/linux/linux-imx.inc
+
+LIC_FILES_CHKSUM = "file://COPYING;md5=bbea815ee2795b2f4230826c0c6b8814"
+
+DEPENDS += "lzop-native bc-native"
+
+SUMMARY = "Linux kernel for Toradex Freescale i.MX based modules"
+
+SRC_URI = "git://git.toradex.com/linux-toradex.git;protocol=git;branch=${SRCBRANCH}"
+
+# Load USB functions configurable through configfs (CONFIG_USB_CONFIGFS)
+KERNEL_MODULE_AUTOLOAD += "${@bb.utils.contains('COMBINED_FEATURES', 'usbgadget', ' libcomposite', '',d)}"
+
+PV:append = "+git${SRCPV}"
+
+LINUX_VERSION = "5.4.129"
+
+SRCREV = "828ec42a6a32004a199f8ee4893fecdc0b15ae92"
+SRCBRANCH = "toradex_5.4-2.3.x-imx"
+
+# Deploy symbols to allow for device tree overlays
+EXTRA_OEMAKE += "DTC_FLAGS=-@ "
+
+# set KBUILD_DEFCONFIG:${MACHINE} = "" and add a file defconfig to SRCURI to
+# inject your defconfig in a bbappend.
+KBUILD_DEFCONFIG:apalis-imx6 ?= "apalis_imx6_defconfig"
+KBUILD_DEFCONFIG:colibri-imx6 ?= "colibri_imx6_defconfig"
+KBUILD_DEFCONFIG:colibri-imx6ull ?= "colibri-imx6ull_defconfig"
+KBUILD_DEFCONFIG:mx7 ?= "colibri_imx7_defconfig"
+KBUILD_DEFCONFIG:mx8 ?= "toradex_defconfig"
```

Comment:
Why not set this in the respective machines?

Reply:
Moved.